kubernetes_asyncio/client/rest.py

```diff
@@ -66,8 +66,8 @@ def __init__(self, configuration, pools_size=4, maxsize=4):
 
         connector = aiohttp.TCPConnector(
             limit=maxsize,
-            ssl_context=ssl_context,
-            verify_ssl=configuration.verify_ssl
+            ssl_context=ssl_context if configuration.verify_ssl else None,
+            verify_ssl=configuration.verify_ssl,
```

Comment:
... and this `,` too. Small and precise change-set is simpler to read and apply. Thank you.